ex2.html

```diff
@@ -1,44 +1,29 @@
 <!DOCTYPE html>
 <html>
 <head>
+	<link rel="stylesheet" type="text/css" href="ex1_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="ex2_header_css.css">
+	<link rel="stylesheet" type="text/css" href="buttons.css">
+
 </head>
 <body>
 
 	<section>
-		<header> 
-			<img src="logo.jpg" style="float:left; width:200px;height:50px;">
-			<h2><a href="ex2.html" target="_self" style="float: left; margin-left: 20px; margin-right: 20px;">Evaluations</a></h2>
-			<h2><a href="ex3.html" target="_self""> New Evaluation</a></h2>
-			<h4><a href="ex1.html" target="_top" style="float: right;">Log out</a></h4>
+		<header class="header"> 
+			<img src="logo.jpg" style="float:left; width:200px;height:40px; margin-left: 3%">
+			<h2><a href="ex2.html" target="_self" class="button1" style="margin-left: 7%; margin-right: 0.1%;">Evaluations</a></h2>
```

Comment:
As a best practice, all heading levels leading up to the current one should be included in the document's markup, in order to provide semantic meaning to the document. That means that in order to have an `<h4>` in your page, you'll also want to have an `<h3>` somewhere above it, with an `<h2>` somewhere above that, and an `<h1>` somewhere above that. If you're only picking certain heading levels because of their default styling, you're doing it wrong. :P